Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'sprockets', '~> 3'
 gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
+gem 'email_validator', '1.6.0'
```

Comment:
is this packaged in all supported distros? if not, I suppose you'd have to package it first
https://packages.debian.org/search?suite=all&section=all&arch=any&searchon=names&keywords=email_validator
http://packages.ubuntu.com/search?suite=xenial&section=all&arch=any&keywords=email_validator&searchon=all